y-octo-utils/bin/doc_merger.rs

```diff
@@ -0,0 +1,100 @@
+use std::{
```

Comment:
what's this for?

Reply:
test the memory usage in loading a large doc or debug doc merge problem